readme.md

```diff
@@ -43,7 +51,7 @@ To configure the plugin:
 
 Bitbucket Server instances are added and configured at the system level. Once they’re added users can select them from the SCM when creating a Jenkins job. You must add at least one Bitbucket Server instance to Jenkins.
 
-When adding a Bitbucket Server instance you must add at least one Bitbucket Server [personal access token](https://confluence.atlassian.com/display/BitbucketServer/personal+access+tokens). Doing this allows users to automatically set up build triggers when creating a Jenkins job. For this to work the tokens you add must have project admin permissions.
+When adding a Bitbucket Server instance you must add at least one Bitbucket Server [personal access token](https://confluence.atlassian.com/display/BitbucketServer/personal+access+tokens) and all tokens added must have project admin permissions. Doing this allows users to automatically set up build triggers when creating a Jenkins job.
```

Comment:
Maybe:
> When adding a Bitbucket Server instance you must add at least one Bitbucket Server [personal access token](https://confluence.atlassian.com/display/BitbucketServer/personal+access+tokens), **configured with project admin permissions**. Doing this allows users to automatically set up build triggers when creating a Jenkins job.
The "all tokens" feels a bit awkward in context, given the preceding sentence refers to them in the singular.